FutureNorthants/northants-website
# Fixes #121 - empty service alert titles

**Testing**
1. Edit a service landing page e.g. _School admissions_ to add an alert.
2. Fill in the alert text but ensure the title is blank and tick the _enable alert_ box.
3. Save and Publish the page.
4. View the page in the frontend.
Without this fix, should 500 error on you. With this fix, should display correctly including the title-less alert.

5. Also look at service pages under the service landing page and expect to see alert correctly.
6. Other tests - try with a non-blank title in the alert, try a disabled alert, remove the alert entirely, expect no errors in any variation.
